classes/WP_CLI/ActionScheduler_WPCLI_Clean_Command.php

```diff
@@ -81,7 +81,7 @@ public function clean( $args, $assoc_args ) {
 	 *
 	 * @param int $batches_processed Number of batches processed.
 	 */
-	protected function print_total_batches( int $batches_processed ) {
+	protected function print_total_batches( $batches_processed ) {
```

Comment:
We don't want to remove these. We'll [update the PHPCS config](https://github.com/woocommerce/action-scheduler/pull/1091) instead.

Reply:
@barryhughes Well that's unfortunate, to have wasted that time. Will kill this PR and branch and start again.

Reply:
Why kill the branch? There are, as far as I could see, only two instances of type hints being removed.

Reply:
Do bear in mind that these overlapping PRs with large changesets also come at a cost to us.

Reply:
Because it's likely there are other rules for PHP 7.